pkg/meta/model/job.go

```diff
@@ -490,39 +490,51 @@ func (job *Job) FillFinishedArgs(args FinishedJobArgs) {
 	args.fillFinishedJob(job)
 }
 
+func marshalArgs(jobVer JobVersion, args []any) (json.RawMessage, error) {
+	if jobVer <= JobVersion1 {
+		rawArgs, err := json.Marshal(args)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		return rawArgs, nil
+	}
+
+	intest.Assert(jobVer == JobVersion2, "job version is not v2")
+	var arg any
+	if len(args) > 0 {
+		intest.Assert(len(args) == 1, "Job.Args should have only one element")
+		arg = args[0]
+	}
+
+	rawArgs, err := json.Marshal(arg)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	return rawArgs, nil
```

Comment:
```suggestion
	return rawArgs, errors.Trace(err)
```

Reply:
can it be passed by lint-check? I can try it.

Reply:
modify it.